libs/order/state/src/order-state.module.ts

```diff
@@ -10,6 +12,7 @@ import {
 
 @NgModule({
   imports: [
+    DaffCartStateModule,
```

Comment:
Is this intentional?

Reply:
yes, order state has a direct dep on cart state